app/views/users/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for @user do |f| %>
```

Comment:
Rails sẽ dùng POST (for new user) nếu new_record? trả về true, dùng PATCH (for update user) nếu trả về false